src/Share.ts

```diff
@@ -19,8 +19,8 @@ class Share {
 
   toJSON(): StringifiedType {
     return {
-      share: this.share.toString("hex"),
-      shareIndex: this.shareIndex.toString("hex"),
+      share: this.share.toString("hex", 64),
```

Comment:
check padding here

Reply:
This function is only used in import key flow and since import flow was broken previously no body is using it. I noticed that non padded shares were causing issues so i added this padding intentionally. I think this wont affect anyone since no body is using this flow so far in prod.